README.md

```diff
@@ -30,47 +30,47 @@ Supported systems are:
 
 ## How to play
 ### Windows
-1. If not already done install Gothic. OpenGothic comes with auto-path detection if your Gothic files are in a common path.
+1. If not already done, install Gothic 2. OpenGothic comes with automatic path detection if your Gothic files are in a common path.
     * "C:\Program Files (x86)\JoWooD\Gothic II"
     * "C:\Gothic II"
     * "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
-2. Download OpenGothic and extract into a folder of your choice. Available options are:
+2. Download OpenGothic and extract it into a folder of your choice. Available options are:
     * A [Pre-Release](https://github.com/Try/opengothic/releases/latest) (recommended)
     * Alternatively a recent test build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master)
 3. Run `Gothic2Notr.exe`.
 
-   If nothing happens check `log.txt` and look for the line `invalid gothic path`. In this case OpenGothic fails to find your Gothic installation and you have to explicitly specify its location via `-g` paramter. Either you create a shortcut to `Gothic2Notr.exe` and change the target line in Properties to e.g.
+   If nothing happens, check `log.txt` and look for the line `invalid gothic path`. In this case OpenGothic fails to find your Gothic installation and you have to explicitly specify its location via `-g` parameter. Either you create a shortcut to `Gothic2Notr.exe` and change the target line in Properties to e.g.
 
    `Gothic2Notr.exe -g "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"`
 
    or you can edit `Gothic2Notr.bat` and run this file instead.
 
 ### Linux
-1. If not already done install Gothic via Wine/Proton or copy the game files from a Windows installation.
-2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and extract into a folder of your choice. Alternatively OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a 3rd party package.
+1. If not already done, install Gothic 2 via Wine/Proton or copy the game files from a Windows installation.
+2. You can download a build from [CI](https://ci.appveyor.com/project/Try/opengothic/history?branch=master) and extract into a folder of your choice. Alternatively, OpenGothic can be built manually. For Arch the [AUR](https://aur.archlinux.org/packages/opengothic) provides a third-party package.
 
-3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use path to your Gothic installation instead)
+3. Run `Gothic2Notr.sh -g "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"` (example path, use the path to your Gothic 2 installation instead)
```

Comment:
I'm not sure about `the path`. While I'm not great at English, `the` should be something very concrete and "path to gothic2" is somewhat general thing (different for different machines).

Reply:
I'd argue on a semantic level it is a very specific path, as indicated by "to". From a developer perspective, it's also a unique path on an individual system.